src/components/Card/Card.st.css

```diff
@@ -50,7 +50,7 @@
     width: 100%;
     height: 100%;
     background-color: value(cardBGColor);
-    -st-mixin: withBorder(
+    -st-partial-mixin: withBorder(
```

Comment:
I think that partial should be used only in the case of extending component and not every mixins.
In this example, this is the class:
```
.withBorder {
    border-style: solid;
    border-color: value(BorderColor);
    border-width: value(BorderWidth);
}
```

when using partial-mixin, it will drop the `border-style` property because it is not affected by the variables.

so this one should be a regular `-st-mixin`

Reply:
Correct 👍